api/ReadableStreamDefaultController.json

```diff
@@ -36,10 +36,10 @@
             "version_added": false
           },
           "samsunginternet_android": {
-            "version_added": "6.0"
+            "version_added": false
```

Comment:
Why false here?

Reply:
The latest Samsung Internet release is based on Chrome 87.

Reply:
Aha, thanks!